main_test.go

```diff
@@ -597,7 +597,7 @@ func TestSignerRevokeApproval(t *testing.T) {
 	})
 }
 
-func TestAddSignerAction(t *testing.T) {
+func TestAddSignerUpdateThresholdAction(t *testing.T) {
```

Comment:
Do we still have the possibility to call them separately? If we do, then we need both test cases. 
And some negative scenarios too